Makefile

```diff
@@ -1,4 +1,5 @@
 
+BROWSERIFY := node_modules/.bin/browserify
```

Comment:
Nah, the number of paths explicitly stated is small enough that it doesn't matter, although I'm curious what your issues were.